Readme.md

```diff
@@ -1,5 +1,21 @@
 ## Use of LLM to improve plane maintenance
 
+# Data Processor
+
+
+### Parameters
+
+- `config` (`data_config` or `None`): The configuration object containing parameters for data processing. If `None`, a `CopaError` will be raised.
+- `base_path` (`str`, optional): The base path where the data is located. Defaults to an empty string, which means the data is located in the current directory.
```

Comment:
Shouldn't we state where the program should be executed from? Alternatively, if base_path is relative, relative to what?